spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -1288,6 +1288,40 @@ for (Pageable p = Pageable.ofSize(25).sortBy(Sort.desc("yearBorn"),
 }
 ----
 
+===== Example with Before/After Cursor
+
+In this example, the application uses a cursor to request pages in forward and previous direction from a specific value, which is the price for a matching product.
+
+[source,java]
+----
+@Repository
+public interface Products extends CrudRepository<Product, Long> {
+  KeysetAwareSlice<Product> findByNameContains(String namePattern, Pageable pageRequest);
+}
+----
+
+Obtaining the next 10 products that cost $50.00 or more:
+
+[source,java]
+----
+float priceMidpoint = 50.0f;
+Pageable pageRequest = Pageable.ofSize(10)
+                               .sortBy(Sort.asc("price"), Sort.asc("id"))
+                               .afterKeyset(priceMidpoint, 0L);
```

Comment:
❓ what is this the parameter "0L"?

Reply:
0L is the value of the id.  The first parameter value, priceMidpoint, corresponds to the first sort (price), and the second parameter value, 0L, corresponds to the second sort (id).